scripts/ccf/sandbox_local/down.sh

```diff
@@ -7,7 +7,7 @@ ccf-sandbox-local-down() {
     set -e
 
     REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../..")"
-    docker compose -f $REPO_ROOT/services/docker-compose.yml down ccf-sandbox --remove-orphans
+    docker compose -f $REPO_ROOT/services/docker-compose.yml down --remove-orphans
 
     unset KMS_URL
```

Comment:
why do we unset env vars when stopping containers? They will be set in the up commands.

Reply:
The environment variables that are set when we bring up a service describe that instance of that service, when we bring that service down those variables are no longer applicable because the instance of that service no longer exists, so in order to avoid mistakes we should unset them.